src/ai/backend/client/cli/ssh.py

```diff
@@ -13,8 +13,8 @@ def ssh(sess_name, port):
     \b
     USER: Name of the user at the session image. Default: work
```

Comment:
This option seems to not exist, and it is not required since Backend.AI sessions always have the `work` user.